.github/workflows/cut-rc.yml

```diff
@@ -244,22 +242,46 @@ jobs:
             echo ":arrow_right: Close the newly created milestone [${{needs.variables.outputs.next-milestone}}](${{github.event.milestone.html_url}}) when ready." >> $GITHUB_STEP_SUMMARY
           fi
 
-  pre-release:
-    name: Publish a GitHub pre-release
-    needs: [variables, cut-rc]
-    runs-on: ubuntu-latest
-    steps:
+      # - name: Run Gosec security scanner
+      #   uses: securego/gosec@master
+      #   with:
+      #     args: "-no-fail -out ${{ steps.tag.outputs.report-file-name }} ./..."
+      # - run: |
+      #     cat <<EOF >> $GITHUB_STEP_SUMMARY
+      #     # gosec Report
+
+      #     \`\`\`
+      #     EOF
+      #     cat "$GITHUB_WORKSPACE/${{ steps.tag.outputs.report-file-name }}" >> $GITHUB_STEP_SUMMARY
+      #     echo '```' >> $GITHUB_STEP_SUMMARY
+
       - name: Create GitHub Pre-release
         id: pre-release
         if: env.DRY_RUN == 'false'
+        env:
+          GH_TOKEN: ${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}
         run: |
-          _MD_SECTION="${{needs.variables.outputs.release-patch}}"
+          set -uo pipefail
+          gh api -H "$ACCEPT_RAW" "${{env.script_url}}" | bash -s -- \
+            draft-release-notes \
+            "${{ needs.variables.outputs.release-patch }}" \
+            "${{ needs.variables.outputs.branch }}"
```

Comment:
I assume this outputs `RELEASE_NOTES_GENERATED.md`, so that is can be used in the next line.
I would suggest to specify the output as a param explicitly in `draft-release-notes` (you may keep this value as a default), as the reader of this code would then be forced to start searching for a function that produced this file and having specified the output here would make this job easier.

Reply:
Another alternative would be to change the `draft-release-notes` to print to stdout. This would force the caller to capture the output and it would make it easier to follow.

Reply:
> Another alternative would be to change the draft-release-notes to print to stdout.

I have tried it and unfortunately didn't work, as it mangled line breaks and the output was in one line. 

The first suggestion makes sense, I will apply it.